remove_unwanted_files.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+export PATH=/bin:/usr/bin
+IFS='
```

Comment:
Why mess with IFS?

Reply:
@mwaggett looks like you might have removed the stuff that was making the IFS hack helpful. Looks like you should be able to remove the IFS change as well, now.

Reply:
I guess I don't really need that now, but when I was testing this out, I was doing `grep -rin` for debugging purposes and I didn't wanted each match on its own line.

Reply:
@e-gris yeah, I kept it around for debugging purposes, but I can take it out now.